tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
```

Comment:
@jtimpe @n0remac  i do not see submission history for ssp files


![1007p1](https://user-images.githubusercontent.com/63075587/216636771-44f785c0-993f-46be-8557-60d88be05e96.PNG)
![1007p2](https://user-images.githubusercontent.com/63075587/216636763-3fd97d03-e1cd-441e-a685-0f2d8302fab6.PNG)

Reply:
Good catch, thank you! I was matching too strictly on the `section`. Fixed and added a test